src/main/java/utils/configurations/Configurations.java

```diff
@@ -0,0 +1,36 @@
+package utils.configurations;
+
+import com.google.common.collect.ImmutableMap;
+import onlineservices.Service;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import services.WindowControl.WindowControlService;
+import utils.filereaders.CsvFileReader;
+
+
+import java.io.File;
+import java.util.ArrayList;
+import java.util.List;
+
+@Configuration
+public class Configurations {
+    private final ImmutableMap<String, Service> allServices = ImmutableMap.of(
+            WindowControlService.class.getName(), new WindowControlService()
+    );
+
+    public List<String> getActivatedServicesNames() {
+        CsvFileReader fileReader = new CsvFileReader();
+        return fileReader.readFile(new File(".\\src\\main\\resources\\ActivatedServices.csv"));
+    }
+
+    @Bean(name = "activeServices")
+    public List<Service> getActivatedServices() {
+        List<Service> activatedServices = new ArrayList<>();
+        List<String> activatesServicesNames = getActivatedServicesNames();
+        for (String serviceName : activatesServicesNames) {
+            Service activatedService = allServices.get(serviceName);
```

Comment:
remove the code from inside of the for loop and add if elses, if the serviceName == WindowControlOnlineService then add windowControlOnlineService() to activatedServices, i also want this logged, if a service is in the list of activated services, i want logged servicex was added to the activated services, if a service is not in the activated services, servicex is not in the activated list ...